plugins/processors/awsapplicationsignals/internal/resolver/attributesresolver_test.go

```diff
@@ -0,0 +1,247 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package resolver
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	semconv "go.opentelemetry.io/collector/semconv/v1.17.0"
```

Comment:
Keep versions consistent.
```suggestion
	semconv "go.opentelemetry.io/collector/semconv/v1.18.0"
```

Reply:
Updated.